alternator/executor.cc

```diff
@@ -96,7 +96,7 @@ std::string make_jsonable::to_json() const {
 }
 
 json::json_return_type make_streamed(rjson::value&& value) {
-    // CMH. json::json_return_type uses std::function, not noncopyable_function. 
+    // CMH. json::json_return_type uses std::function, not noncopyable_function.
```

Comment:
CMH?

Reply:
CMH == code missing here/change missing here. Old habit, basically "todo". The above relating to the fact that we can't use a move-only value in the function in question, due to api.